google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/SpannerPool.java

```diff
@@ -186,6 +187,7 @@ private SpannerPoolKey(ConnectionOptions options) throws IOException {
       this.usePlainText = options.isUsePlainText();
       this.userAgent = options.getUserAgent();
       this.routeToLeader = options.isRouteToLeader();
+      this.enableEndToEndTracing = options.enableEndToEndTracing();
```

Comment:
In all of these methods as well: move down to the other tracing properties

Reply:
Done